specifyweb/context/views.py

```diff
@@ -258,10 +258,7 @@ def user(request):
     data = obj_to_data(request.specify_user)
     data['isauthenticated'] = request.user.is_authenticated
     data['available_collections'] = users_collections(connection.cursor(), request.specify_user.id)
-
-    data['agent'] = obj_to_data(request.specify_user_agent) \
-        if hasattr(request.specify_user_agent, '_meta') \
-        else None
+    data['agent'] = obj_to_data(request.specify_user_agent) if request.specify_user_agent != None else None
```

Comment:
request.specify_user_agent is a Django SimpleLazyObject, but applying an equality operator will force it.